src/model.js

```diff
@@ -182,6 +182,74 @@ export class Seed extends Str {
       }
     }
   }
+
+  /**
+   * encrypt the password.
+   * @param {string} password - The password used when encrypting seed previously.
+   * @param {number} rounds - The number of encryption round when encrypting seed previously.
+   * @returns {string} The encrypted password.
+   */
+  static strengthenPassword(password, rounds) {
+    if (rounds === void 0) { rounds = 5000; }
+    while (rounds--)
+        password = hs.sha256Hash(password);
+    return password.toString();
+  }
+
+  /**
+   * encrypt the seed.
+   * @param {string} password - The password used when encrypting seed.
+   * @param {number} encryptionRounds - The number of encryption round when encrypting seed.
+   * @returns {string} The encrypted seed.
+   */
+  encryptSeed(password, encryptionRounds) {
+    if (typeof this.data !== 'string') {
```

Comment:
This checking can be omitted since `encryptSeed` is an instance method. Data in model `Seed` will be validated on instantiating `Seed`